File: aoj/volume100/java/A10027.java
import java.util.Scanner;

class A10027 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int scoreA = 0;
        int scoreB = 0;

        int nGames = scan.nextInt();
        for (int i = 0; i < nGames; i++) {
            String cardA = scan.next();
            String cardB = scan.next();

            int result = cardA.compareTo(cardB);
            if (result == 0) {
                scoreA += 1;
                scoreB += 1;
            } else if (result > 0) {
                scoreA += 3;
            } else if (result < 0) {
                scoreB += 3;
            }
        }

        System.out.printf("%d %d\n", scoreA, scoreB);
    }
}

File: aoj/volume100/php/A10027.php
<?php

fscanf(STDIN, "%d", $n);

$score_a = 0;
$score_b = 0;
for ($i = 0; $i < $n; $i++) {
    fscanf(STDIN, "%s %s", $card_a, $card_b);
    if ($card_a === $card_b) {
        $score_a += 1;
        $score_b += 1;
    } else if ($card_a > $card_b) {
        $score_a += 3;
    } else if ($card_a < $card_b) {
        $score_b += 3;
    }
}

echo "$score_a $score_b\n";
